services/convert/mirror.go

```diff
@@ -15,8 +15,8 @@ func ToPushMirror(pm *repo_model.PushMirror) (*api.PushMirror, error) {
 		RepoName:       repo.Name,
 		RemoteName:     pm.RemoteName,
 		RemoteAddress:  pm.RemoteAddress,
-		CreatedUnix:    pm.CreatedUnix.FormatLong(),
-		LastUpdateUnix: pm.LastUpdateUnix.FormatLong(),
+		CreatedUnix:    pm.CreatedUnix.AsTime(),
+		LastUpdateUnix: pm.LastUpdateUnix.AsTimePtr(),
```

Comment:
But `pm.LastUpdateUnix` maybe nil.

Reply:
`pm.LastUpdateUnix` is a `timeutil.TimeStamp` not `*timeutil.TimeStamp`. It can only be `pm.LastUpdateUnix.IsZero()` if I'm not mistaken.

Reply:
Yeah, I was also a bit confused about that. So I guess I change it back to not be a pointer?

Reply:
Ah, so maybe we should change 
```go
// AsTimePtrInLocation convert timestamp as *time.Time in customize location
func (ts TimeStamp) AsTimePtrInLocation(loc *time.Location) *time.Time {
	tm := time.Unix(int64(ts), 0).In(loc)
	return &tm
}
```
to 
```go
// AsTimePtrInLocation convert timestamp as *time.Time in customize location
func (ts TimeStamp) AsTimePtrInLocation(loc *time.Location) *time.Time {
  if ts.IsZero() {
      return nil
  }
	tm := time.Unix(int64(ts), 0).In(loc)
	return &tm
}

Reply:
So I guess this is resolved and the proposed change should be done in a different PR?